src/main/flight/pid.c

```diff
@@ -529,8 +529,10 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
                         inCrashRecoveryMode = false;
                     }
                 } else {
+                    if (inCrashRecoveryMode) {
```

Comment:
else if (inCrashRecoveryMode) {
 BEEP_OFF;
 inCrashRecoveryMode = false;
}
to reduce the double branch

Reply:
This PR https://github.com/betaflight/betaflight/pull/4400 could include this modification.

Reply:
Good point. Cosmetic fixes never hurt.

Reply:
@Vidalcris: Done.